.ci/run.py

```diff
@@ -19,7 +19,7 @@
 from conans import __version__ as conan_version
 
 
-FAIL_FAST = os.getenv("FAIL_FAST", "0").lower() in ["1", "y", "yes", "true"]
+FAIL_FAST = os.getenv("FAIL_FAST", "1").lower() in ["1", "y", "yes", "true"]
```

Comment:
Making if fail fast, this ci takes a lot of time to run, I think it's better to know if something does not work first.